dev/com.ibm.ws.concurrent.persistent/src/com/ibm/ws/concurrent/persistent/db/DatabaseTaskStore.java

```diff
@@ -593,7 +593,7 @@ public long findOrCreatePollPartition() throws Exception {
                 // Initialize to already-expired and eligible for any server to claim.
                 // Use a fractional second to help avoid accessing the database around the same time as scheduled tasks
                 // which might be scheduled to run on the hour or minute.
-                partition.EXPIRY = (System.currentTimeMillis() / 1000 - 1) * 1000 + 600;
+                partition.EXPIRY = System.currentTimeMillis() / 1000 * 1000 + 600;
```

Comment:
Just to be sure you are aware, with this change it is still possible that this could be a time in the past.
For example, 25800 / 1000 * 1000 + 600 evaluates to 25600.